onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlGraphFusionHelper.cpp

```diff
@@ -293,8 +311,22 @@ namespace DmlGraphFusionHelper
         for (size_t i = 0; i < graphDesc.nodes.size(); ++i)
         {
             auto& nodeInfo = graphDesc.nodes[i];
-            dmlOperatorGraphNodes[i] = DML_OPERATOR_GRAPH_NODE_DESC{nodeInfo.op.Get(), nodeInfo.name.data()};
-            dmlGraphNodes[i] = DML_GRAPH_NODE_DESC{DML_GRAPH_NODE_TYPE_OPERATOR, &dmlOperatorGraphNodes[i]};
+            
+            if (std::holds_alternative<Microsoft::WRL::ComPtr<IDMLOperator>>(nodeInfo.nodeDef))
+            {
+                dmlOperatorGraphNodes[i] = DML_OPERATOR_GRAPH_NODE_DESC{std::get<Microsoft::WRL::ComPtr<IDMLOperator>>(nodeInfo.nodeDef).Get(), nodeInfo.name.data()};
+                dmlGraphNodes[i] = DML_GRAPH_NODE_DESC{DML_GRAPH_NODE_TYPE_OPERATOR, &dmlOperatorGraphNodes[i]};
+            }
+            else
+            {
+                dmlConstantGraphNodes[i] = DML_CONSTANT_DATA_GRAPH_NODE_DESC_PREVIEW{
+                std::get<std::vector<uint8_t>>(nodeInfo.nodeDef).data(), 
+                std::get<std::vector<uint8_t>>(nodeInfo.nodeDef).size(), 
+                nodeInfo.name.data()
+                };
```

Comment:
Params not indented.

Assigning to temporary would help readability.

```suggestion
                auto nodeDefinitionData& std::get<std::vector<uint8_t>>(nodeInfo.nodeDef);
                dmlConstantGraphNodes[i] = DML_CONSTANT_DATA_GRAPH_NODE_DESC_PREVIEW{
                    nodeDefinitionData.data(),
                    nodeDefinitionData.size(),
                    nodeInfo.name.data()
                };
```

Reply:
The suggested code will also take care of the annoying "Line ends in whitespace" warning :b.

Reply:
I will also use lint runner on WindowsAI branch, which should handle these warnings for multiple commits at once.